### ISSUE TITLE
Keyboard Shortcut to Deselect All Selected Items?

### ISSUE THREAD

**Issue by jpdevries** [jpdevries/eureka#112]
Took a swing at this but it is actually really tricky. Or I'm making it out to be trickier than it should be. Basically I don't want every `<MediaRow>` to re-render whenever one is checked so they manage their checked state internally, and that is what is making it tricky.
